src/postman/__tests__/operation.test.ts

```diff
@@ -0,0 +1,89 @@
+import { HeaderDefinition, RequestBody } from 'postman-collection';
+import { transformPostmanCollectionOperation } from '../operation';
+
+describe('transformPostmanCollectionOperation()', () => {
+  describe('operation can be found', () => {
+    describe('description is set', () => {
+      it('returns operation with description', () => {
+        expect(
+          transformPostmanCollectionOperation({
+            document: {
+              item: [
+                {
+                  request: {
+                    method: 'get',
+                    url: '/path/:param?a=b',
+                    body: { mode: 'raw', raw: 'test' } as RequestBody,
+                    header: [{ key: 'header', value: 'a header' }] as HeaderDefinition,
+                  },
+                  description: 'desc',
+                },
+              ],
+            },
+            method: 'get',
+            path: '/path/{param}',
+          }),
+        ).toEqual({
+          description: 'desc',
+          id: '?http-operation-id?',
```

Comment:
@marbemac Can we get rid of this property? It has been there since forever and it's completely useless at the moment.

Reply:
+1

Reply:
@marbemac ping